rmw_zenoh_cpp/src/detail/zenoh_config.cpp

```diff
@@ -69,6 +73,33 @@ std::optional<zenoh::Config> _get_z_config(
       "Invalid configuration file %s", configured_uri);
     return std::nullopt;
   }
+
+  // Override the zenoh config by key/value pairs.
+  const char * key_val_pairs_str;
+  if (NULL != rcutils_get_env(zenoh_config_override, &key_val_pairs_str)) {
+    // NULL is returned if everything is ok.
+    RMW_ZENOH_LOG_ERROR_NAMED(
+      "rmw_zenoh_cpp", "Envar %s cannot be read.", zenoh_config_override);
+    return std::nullopt;
+  }
+  if (key_val_pairs_str[0] != '\0') {
+    std::stringstream ss(key_val_pairs_str);
+    std::string pair;
+    while (std::getline(ss, pair, ';')) {
+      std::stringstream pair_stream(pair);
+      std::string key, val;
+      if (std::getline(pair_stream, key, '=') && std::getline(pair_stream, val)) {
+        config.insert_json5(key, val, &result);
+        if (result != Z_OK) {
+          RMW_ZENOH_LOG_ERROR_NAMED(
+            "rmw_zenoh_cpp",
+            "Invalid configuration key/value pair: (%s, %s)", key.c_str(), val.c_str());
+          return std::nullopt;
```

Comment:
```suggestion
            "Invalid configuration key/value pair: (%s, %s). Ignoring override...", key.c_str(), val.c_str());
          continue;
```

Reply:
Addressed in [26e0c8b](https://github.com/ros2/rmw_zenoh/pull/551/commits/26e0c8b5e84cd482951b4d33dc3190d211182925).